dot/sync/chain_sync_test.go

```diff
@@ -203,12 +203,30 @@ func TestChainSync_getTarget(t *testing.T) {
 	require.Equal(t, big.NewInt(2<<32-1), cs.getTarget())
 
 	cs.peerState = map[peer.ID]*peerState{
-		"testA": {
-			number: big.NewInt(1000),
+		"a": {
+			number: big.NewInt(-100), //outlier
```

Comment:
nit usually comments without space after `//` are to be machine readable (e.g. `//go:generate`)

```suggestion
			number: big.NewInt(-100), // outlier
```